xml/admin_troubleshooting.xml

```diff
@@ -0,0 +1,259 @@
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Troubleshooting</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker/>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    This chapter summarizes frequent problems that occur while using
+    &productname; and their solutions.
+   </para>
+  </abstract>
+ </info>
+
+ <sect1 xml:id="sec.admin.troubleshooting.overview">
+  <title>Overview</title>
+  <para>
+   This chapter is a collection of frequent problems that are reported
+   for &productname;. Additionally, &suse; support collects problems
+   and their solutions online at <link
+   xlink:href="https://www.suse.com/support/kb/?id=SUSE_CaaS_Platform"
+   />.
+  </para>
+  <para>
+   In case of problems, a detailed system report can be created with
+   the <command>supportconfig</command> command line tool. It will
+   collect information about the system such as: current kernel
+   version, hardware, installed packages, partition setup, and much
+   more. The result is a TAR archive of files. After opening a Service
+   Request (SR), you can upload the TAR archive to Global Technical
+   Support. It will help to locate the issue you reported and to assist
+   you in solving the problem. For details, see <link xlink:href=
+   "https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_adm_support.html"
+   />.
+  </para>
+  <para>
+   While resolving a technical difficulty with SUSE Support, you may
+   receive a so-called Program Temporary Fix (PTF). PTFs may be issued
+   for various packages as RPMs. For details about handling PTFs, see
+   <xref linkend="sec.admin.software.patch" />.
+  </para>
+  <para>
+   For details about how to contact &suse; support, refer to <link
+   xlink:href=
+   "https://www.suse.com/media/flyer/suse_customer_support_quick_reference_guide_flyer.pdf"
+   /> and <link xlink:href="https://www.suse.com/support/" />.
+  </para>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.troubleshooting.failed_bootstrap">
+  <title>Recover from Failed Bootstrap</title>
+  <para>
+   
+  </para>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.troubleshooting.failed_update">
+  <title>Recovering from Failed Update</title>
+  <para>
+   See <xref
+   linkend="sec.admin.software.transactional-updates.recovering" />.
+  </para>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.troubleshooting.etcd">
+  <title>Checking <literal>etcd</literal> Health</title>
+  <para>
+   To work with <command>etcdctl</command>, you have to pass parameters
+   with paths to required certificates.
+  </para>
+  <para>
+   Use SSH to log into one of the cluster nodes, for example your first master.
+   The following command provides an example for using <command>etcdctl</command>.
+  </para>
+<screen>&prompt.root;<command>source /etc/sysconfig/etcdctl</command>
```

Comment:
Before this 
set -a`
then 
`source /etc/sysconfig/etcdctl` and finally `etcdctl cluster-health`